src/pages/CategoryPage/CategoryPage.tsx

```diff
@@ -159,7 +160,9 @@ export const CategoryPage = () => {
 
             <TabsCategory subcategories={cat?.subCategories ?? []} />
 
-            {recipesData?.data && <RecipeList recipes={recipesData.data} gridVariant='low' />}
+            {!isLoading && !isFetching && recipesData?.data && recipesData?.data?.length > 0 && (

```

Comment:
как минимум вот это !isLoading && !isFetching можно убрать, так как эти условия у тебя обрабатываются выше

Reply:
```suggestion
            {recipesData?.data?.length > 0 && (

```
вот так должно тоже работать